src/crawler/scraper.py

```diff
@@ -24,16 +24,96 @@
 from enum import Enum
 
 
+class Count:
+    def __init__(self, name_of_app, package_name,
+                 country, lang):
+        self.name_of_app = name_of_app
+        self.package_name = package_name
+        self.lang = lang
+        self.country = country
+
+    def __get_file_name(self, score):
+        file_name = 'review_' + self.package_name + \
+            '_' + self.country + '_'+self.lang+'_'+str(score) + '.csv'
+        return file_name
+
+    def __get_count_file_name(self, duration):
+        file_name = duration + '_review_count_' + self.package_name + \
+            '_' + self.country + '_'+self.lang+'_' + '.csv'
+        return file_name
+
+    def __should_count(self, duration):
+        file = self.name_of_app + '//' + self.__get_count_file_name(duration)
+        return Utils.is_file_present(file)
+
+    def count_reviews_each_month(self):
+        if self.__should_count('month'):
+            return
+        dir_name = self.name_of_app
+        file_name = self.__get_file_name(1)
+        path = ".\\" + dir_name + "\\" + file_name
+        df = pd.read_csv(path)
+        for score in range(2, 6):
+            file_name = self.__get_file_name(score)
+            path = ".\\" + dir_name + "\\" + file_name
+            df1 = pd.read_csv(path)
+            df = df.append(df1)
+        df['at'] = pd.to_datetime(df['at'])
+        result = df.groupby([df['at'].dt.year, df['at'].dt.month])
+        # df.index = pd.to_datetime(df['at'],format='%m/%d/%y %I:%M%p')
+        # df=df.groupby(by=[df.index.month, df.index.year])
+        df.sort_values(by=['at'], ascending=True, inplace=True)
+        df1 = df.groupby([df['at'].dt.year, df['at'].dt.month]).agg({'count'})
+        l1 = df1[('at', 'count')].tolist()
```

Comment:
rename this variable to header

Reply:
l1 is not the header. It contains the data of number of reviews. 
Have updated the variable name with no_of_reviews